driver/razeraccessory_driver.c

```diff
@@ -142,6 +142,10 @@ static ssize_t razer_attr_read_device_type(struct device *dev, struct device_att
         device_type = "Razer Core\n";
         break;
 
+    case USB_DEVICE_ID_RAZER_LAPTOP_STAND_V1_CHROMA:
+        device_type = "Razer Laptop Stand V1 Chroma\n";
```

Comment:
Is the name Razer uses not just "Razer Laptop Stand Chroma"?

Reply:
It is, but there is a V2 and the two stands are not easily identifiable from a picture.